src/webui/src/components/Sessionpro.tsx

```diff
@@ -138,16 +138,28 @@ class Sessionpro extends React.Component<{}, SessionState> {
                     const searchSpace = JSON.parse(sessionData.params.searchSpace);
                     Object.keys(searchSpace).map(item => {
                         const key = searchSpace[item]._type;
-                        if (key === 'loguniform' || key === 'qloguniform') {
-                            let value = searchSpace[item]._value;
-                            const a = Math.pow(10, value[0]);
-                            const b = Math.pow(10, value[1]);
-                            if (a < b) {
-                                value = [a, b];
-                            } else {
-                                value = [b, a];
-                            }
-                            searchSpace[item]._value = value;
+                        let value = searchSpace[item]._value;
+                        switch (key) {
+                            case 'loguniform':
+                            case 'qloguniform':
+                                const a = Math.pow(10, value[0]);
+                                const b = Math.pow(10, value[1]);
+                                if (a < b) {
+                                    value = [a, b];
+                                } else {
+                                    value = [b, a];
```

Comment:
Maybe we should not change the order of the values specified by user in search space?

Reply:
ok